src/patternfly/base/_variables.scss

```diff
@@ -290,3 +290,7 @@
 @include pf-v5-theme-dark {
   @include pf-v5-theme-dark-variables;
 }
+
+@import "./tokens/tokens-palette";
+@import "./tokens/tokens-default";
+@import "./tokens/tokens-dark";
```

Comment:
Oh an extra line after this last line will make the little red icon go away. We generally add an extra line at EOF to all things, so if you were to import this file and another file right after it it would import as...

```
[...]
@import "./tokens/tokens-palette";
@import "./tokens/tokens-default";
@import "./tokens/tokens-dark";
// other file import would start here
```

insetead of

```
[...]
@import "./tokens/tokens-palette";
@import "./tokens/tokens-default";
@import "./tokens/tokens-dark";// other file import would start here
```